src/config/test_config.py

```diff
@@ -2,9 +2,9 @@
 # Do remember to revert it back to False before merging any PR (including Benchmark dedicated PR)
 ENABLE_BENCHMARK_DEV_MODE = False
```

Comment:
Please set ENABLE_BENCHMARK_DEV_MODE = true to test out the benchmark changes.

Reply:
Addressed